client/src/librarySymbolManager.ts

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as LC from "vscode-languageclient/node";
+import * as path from "path";
+import * as vscode from "vscode";
+
+import { LibraryJson, PowerQueryApi } from "./vscode-powerquery.api";
+import { Convert } from "./libraryExportJSON";
+
+const ErrorMessagePrefix: string = "Error processing symbol directory path. Please update your configuration.";
```

Comment:
Consider putting these constants under LibrarySymbolManager as they seemed to be used exclusively within there